src/pages/developers/learning-tools.tsx

```diff
@@ -45,77 +44,75 @@ import SpeedRunEthereumImage from "@/public/images/dev-tools/speed-run-ethereum.
 import TenderlyImage from "@/public/images/dev-tools/tenderly.png"
 import EnterpriseEth from "@/public/images/enterprise-eth.png"
 
-const Page = (props: ChildOnlyProp) => (
-  <Flex
-    direction="column"
-    align="center"
-    w="full"
-    mx="auto"
-    mt={16}
-    mb={0}
+const Page = ({ className, ...props }: BaseHTMLAttributes<HTMLDivElement>) => (
+  <div
+    className={cn(
+      "mx-auto mb-0 mt-16 flex w-full flex-col items-center",
+      className
+    )}
     {...props}
   />
 )
 
-const Header = (props: ChildOnlyProp) => (
-  <Flex
-    as="header"
-    direction="column"
-    align="center"
-    textAlign="center"
-    maxW="896px"
-    py={0}
-    px={8}
-    m="auto"
+const Header = ({ className, ...props }: BaseHTMLAttributes<HTMLElement>) => (
+  <header
+    className={cn(
+      "m-auto flex max-w-[896px] flex-col items-center px-8 py-0 text-center",
+      className
+    )}
     {...props}
   />
 )
 
-const H1 = (props: ChildOnlyProp) => (
-  <Heading
-    as="h1"
-    color="text"
-    fontStyle="normal"
-    fontFamily="monospace"
-    textTransform="uppercase"
-    fontWeight="semibold"
-    fontSize="2rem"
-    lineHeight={1.4}
-    textAlign="center"
+const H1 = ({
+  className,
+  ...props
+}: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <h1
+    className={cn(
+      "text-text text-center font-mono text-2xl font-semibold uppercase leading-[1.4]",
+      className
+    )}
     {...props}
   />
 )
 
-const Subtitle = (props: HeadingProps) => (
-  <Heading
-    fontSize="xl"
-    lineHeight={1.4}
-    fontWeight="normal"
-    color="text300"
-    maxW="55ch"
-    mb={2}
-    mt={4}
+const Subtitle = ({
+  className,
+  ...props
+}: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <h2
+    className={cn(
+      "text-text300 mb-2 mt-4 max-w-[55ch] text-xl font-normal leading-[1.4]",
+      className
+    )}
     {...props}
   />
 )
 
-const SubtitleTwo = (props: ChildOnlyProp) => <Subtitle mt={0} {...props} />
+const SubtitleTwo = ({
+  className,
+  ...props
+}: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <Subtitle className={cn("mt-0", className)} {...props} />
+)
 
-const ContentBox = (props: ChildOnlyProp) => (
-  <Box py={4} px={8} w="full" {...props} />
+const ContentBox = ({
+  className,
+  ...props
+}: BaseHTMLAttributes<HTMLDivElement>) => (
+  <div className={cn("w-full px-8 py-4", className)} {...props} />
 )
 
-const StackContainer = (props: ChildOnlyProp) => (
-  <Box
-    border="1px"
-    borderColor="border"
-    borderRadius={{ base: 0, sm: "base" }}
-    w={{ base: "full", sm: "96%" }}
-    mx={{ base: 0, sm: 8 }}
-    my={8}
-    px={8}
-    py={12}
-    background="ednBackground"
+const StackContainer = ({
+  className,
+  ...props
+}: BaseHTMLAttributes<HTMLDivElement>) => (
+  <div
+    className={cn(
+      "sm:rounded-base mx-0 my-8 w-full rounded-none border border-border bg-ednBackground px-8 py-12 sm:mx-8 sm:w-[96%]",
```

Comment:
lets use
```suggestion
      "sm:rounded-base mx-0 my-8 w-full rounded-none border border-border bg-background-highlight px-8 py-12 sm:mx-8 sm:w-[96%]",
```
and we avoid the inclusion of this old color in the tw config